<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDialog</class>
 <widget class="QDialog" name="mainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapbender QGIS-Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="addSectionComboBoxLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Please fill the server configuration fields</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="newServiceNameLabel">
       <property name="text">
        <string>Service name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="newServiceNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="newServerAddressLabel">
       <property name="text">
        <string>Server address</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="newServerAddressLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="newServerPortLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="newServerPortLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="newUserNameLabel">
       <property name="text">
        <string>User name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="newUserNameLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="newPasswordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="newPasswordLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="saveNewSectionButton">
     <property name="text">
      <string>Save new section</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="addSectionDialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
